datadog_lambda/wrapper.py

```diff
@@ -254,13 +254,7 @@ def _inject_authorizer_span_headers(self, request_id):
         injected_headers = {}
         source_span = self.inferred_span if self.inferred_span else self.span
         span_context = source_span.context
-        injected_headers[TraceHeader.TRACE_ID] = str(span_context.trace_id)
-        injected_headers[TraceHeader.PARENT_ID] = str(span_context.span_id)
-        sampling_priority = span_context.sampling_priority
-        if sampling_priority is not None:
-            injected_headers[TraceHeader.SAMPLING_PRIORITY] = str(
-                span_context.sampling_priority
-            )
+        propagator.inject(span_context, injected_headers)
```

Comment:
Great catch, I wonder if this methods should eventually be refactored into a tracing service and not the wrapper 😕

Reply:
the `propagator.inject` ends up injecting into `injected_headers`, correct?

Reply:
Good question! 

Having the same thought, I went ahead trying to refactor it. 
Then I found it's already in a helper method called `_inject_authorizer_span_headers`.
Then it felt that the `injection` is more relevant to the `wrapper` because the wrapper now has the `inferred_span` or `span` data and it also should be responsible for the final `response` data. And then at this point, the `injection` became more like `prepare_response` and the only tracer related part is already encapsulated by the `propagator.inject` helper.
Hope this makes sense.